index.html

```diff
@@ -1,160 +1,278 @@
             <h3 class="works_title">Tonic</h3>
             <ul class="works_options">
-                <li><a href="#" class="works_list1">CANOPY</a></li>
-                <li><img src="./image/Counter.png" alt=""></li>
-                <li><a href="#" class="works_list">Back End Dev</a></li>
-                <li><img src="./image/Counter.png" alt=""></li>
-                <li><a href="#" class="works_list">2015</a></li>           
+              <li><a href="#" class="works_list">CANOPY</a></li>
+              <li><img class="counter" src="./image/Counter.png" alt="" /></li>
+              <li><a href="#" class="works_list">Back End Dev</a></li>
+              <li><img class="counter" src="./image/Counter.png" alt="" /></li>
+              <li><a href="#" class="works_list">2015</a></li>
             </ul>
-            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p class="works_parragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
             <ul class="languages">
-                <li class="lang_skill">html</li>
-                <li class="lang_skill">css</li>
-                <li class="lang_skill">javaScript</li>
+              <li class="lang_skill">html</li>
+              <li class="lang_skill">css</li>
+              <li class="lang_skill">javaScript</li>
             </ul>
             <a href="#" class="button">See Project</a>
+
+          </div>
+          
+
         </div>
-        <div class="works_card">
-            <img class="works_img" src="./image/works_2.png" alt="works_img">
-            <h3 class="works_title">Multi-Post<br>Stories</h3>
+        <div class="works_card-1">
+          <img class="works_img" src="./image/works_2.png" alt="works_img" />
+          <img class="works_img_desktop" src="./image/desktop/card2.png" alt="works_img_desktop">
+          <div class="sub-works_card">
+            <h3 class="works_title">Multi-Post<br />Stories</h3>
             <ul class="works_options">
-                <li><a href="#" class="works_list1">CANOPY</a></li>
-                <li><img src="./image/Counter.png" alt=""></li>
-                <li><a href="#" class="works_list">Back End Dev</a></li>
-                <li><img src="./image/Counter.png" alt=""></li>
-                <li><a href="#" class="works_list">2015</a></li>           
-            </ul>
-            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <li><a href="#" class="works_list1">CANOPY</a></li>
+              <li><img class="counter" src="./image/Counter.png" alt="" /></li>
+              <li><a href="#" class="works_list">Back End Dev</a></li>
+              <li><img class="counter" src="./image/Counter.png" alt="" /></li>
+              <li><a href="#" class="works_list">2015</a></li>
```

Comment:
- [ ]  it would be better to place these tags in desktop version  as in the Figma design in order to match it 
- Yours 
<img width="280" alt="Screen Shot 2022-10-01 at 22 19 49" src="https://user-images.githubusercontent.com/61790587/193428756-fc0d0919-a3c8-4904-b929-2b8d67483a9f.png">
-Figma
<img width="340" alt="Screen Shot 2022-10-01 at 22 20 01" src="https://user-images.githubusercontent.com/61790587/193428764-e0102819-d364-4fd1-9e7e-0799abeceeba.png">